modules/manage/partials/iceberg/about-iceberg-topics.adoc

```diff
@@ -150,6 +150,28 @@ SUBJECT          VERSION   ID   TYPE
 ----
 
 ifdef::env-cloud[]
+For Azure clusters, you will need to add subnet IDs from the REST catalog service or other clients requiring access to the Iceberg data to your cluster's allow list.
+
+Make a xref:api:ROOT:cloud-controlplane-api.adoc#patch-/v1/clusters/-cluster.id-[`PATCH /v1/clusters/<cluster-id>`] request to add the subnet IDs to the allow list:
+
+[,bash]
+----
+curl -X PATCH https://api.cloud.redpanda.com/v1/clusters/<cluster-id> \
+  -H "Content-Type: application/json" \
+  -H "Authorization: Bearer ${RP_CLOUD_TOKEN}" \
+  -d @- << EOF
+{
+  "cloud_storage": {
```

Comment:
@simon0191 just want to double check the schema of the request body here, does `cloud_storage` need to be nested under some parent field?